src/lib/layout/aem-allowed-components-container/aem-allowed-components-container.component.ts

```diff
@@ -11,7 +11,7 @@
  */
 
 import { Component, Input } from '@angular/core';
-import { AEMContainerComponent } from '../aem-container/aem-container.component';
+import {AEMContainerComponent, AEMContainerComponentProperties} from '../aem-container/aem-container.component';
```

Comment:
formatting: retain a space distance between { and the text
so:
{ AEMContainerComponent, AEMContainerComponentProperties }